README.md

```diff
@@ -670,7 +677,7 @@ aem {
 ```
 
 When building via command `gradlew :app:build`, then the effect will be a CRX package with assembled JCR content and OSGi bundles from projects: `:app:core`, `:app:common` only.
-
+// as I can see gradle-aem-multi structure has changed and is not aligned with does anymore
```

Comment:
yep, now it is describing sth virtual, maybe better too describe multi as you are suggesting